lib/smart_answer/calculators/benefit_cap_calculator_configuration.rb

```diff
@@ -37,6 +37,10 @@ def descriptions(version)
       end
     end
 
+    def all_questions
+      dataset.keys.inject({}){|versions, version| versions.merge(questions(version))}
```

Comment:
Minor: I'd prefer some more spaces in here i.e. like this:

```ruby
dataset.keys.inject({}) { |versions, version| versions.merge(questions(version)) }
```

I suspect Rubocop will complain about this anyway.